{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "7b50ed9d_a9d9a064",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1121838
      },
      "writtenOn": "2023-10-05T20:03:06Z",
      "side": 1,
      "message": "The bug is due to b/303035093",
      "revId": "7c0ac1e7215deb05f20b4c9924cb49397e3e2766",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}